libs/openFrameworks/graphics/ofGraphics.cpp

```diff
@@ -398,11 +398,11 @@ void ofBackgroundGradient(const ofColor& start, const ofColor& end, ofGradientMo
 		gradientMesh.addVertex(glm::vec3(center, 0.f));
 		gradientMesh.addColor(start);
 		int n = 32; // circular gradient resolution
-		float angleBisector = TWO_PI / (n * 2);
+		float angleBisector = glm::two_pi<float>() / (n * 2);
```

Comment:
Do all of the numbers need to be floats? Same for line 405 in ofGraphics.cpp
float angleBisector = glm::two_pi<float>() / ((float)n * 2.0f);